src/icons/ArrowRight/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const ArrowRight = ({ colour }) => (
```

Comment:
may be good practice to pass rest props to the svg node on all svgs `({ colour, ...rest })`

``` <svg
     version="1.1"
     xmlns="http://www.w3.org/2000/svg"
     x="0px"
     ...
     viewBox="0 0 17 13"
     {...rest}
   >```